src/components/dialogs/inputs/input-hooks.js

```diff
@@ -864,3 +736,63 @@ export const useCSVReader = ({ label, header }) => {
     }, [selectedFile, intl, header]);
     return [selectedFile, setSelectedFile, field, fileError];
 };
+
+export const useRadioValue = ({
+    label,
+    possibleValues = [],
+    defaultValue,
+    id,
+    validation = {},
+    inputForm,
+    doTranslation = true,
+}) => {
+    const [value, setValue] = useState(defaultValue);
+    const intl = useIntl();
+
+    useEffect(() => {
+        // Updates the component when the correct default value is given by the parent component.
+        if (defaultValue?.length > 0) {
+            setValue(defaultValue);
+        }
+    }, [defaultValue]);
+
+    useEffect(() => {
+        function validate() {
+            return true;
+        }
+
+        inputForm.addValidation(id ? id : label, validate);
+    }, [label, validation, inputForm, value, id]);
+
+    const field = useMemo(() => {
+        return (
+            <FormControl>
+                {label && (
+                    <FormLabel id={id ? id : label}>
+                        <FormattedMessage id={label} />
+                    </FormLabel>
+                )}
+                <RadioGroup
+                    row
+                    aria-labelledby={id ? id : label}
+                    value={value ?? defaultValue}
+                    onChange={(e) => setValue(e.target.value)}
```

Comment:
const handleChangeValue = useCallback(
(event) => {
setValue(event.target.value)
inputForm.setHasChanged(true);
},
[inputForm]
);
and use it in onChange prop.

So the validate button is enable when the user change the radio button state

Reply:
Yes, OK.